crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -479,7 +479,7 @@ const transform = (
         if (!result.result) return reject(new Error("No result from loaders"));
         const [source, map] = result.result;
         resolve({
-          source,
+          source: Buffer.isBuffer(source) ? source.toString("utf8") : source,
```

Comment:
The result might be some binary data that is not an utf-8 string.

E. g. some image-processing-loader can return an image.